tutorials/developers/dev_pipeline/dev_pipeline.tex

```diff
@@ -0,0 +1,62 @@
+\documentclass{tufte-handout}
+\usepackage{../braph2_dev}
+%\geometry{showframe} % display margins for debugging page layout
+
+\title{Implement a new Pipeline}
+
+\author[The BRAPH~2 Developers]{The BRAPH~2 Developers}
+
+\begin{document}
+
+\maketitle
+
+\begin{abstract}
+\noindent
+This is the developer tutorial for implementing a new pipeline. 
+In this tutorial, we will explain how to create a \fn{*.braph2} file to change the steps of a pipeline in braph2genesis. Here, we will use as an example the pipeline \code{Pipeline Structural Multiplex Comparison BUD} (the pipeline for comparing two groups of subjects using structural multiplex data and binary undirected graphs at fixed densities).
+\end{abstract}
```

Comment:
In this tutorial, we will explain how to edit a \fn{*.braph2} file to change the steps of a pipeline in braph2genesis. Here, we will use as an example the pipeline \code{Pipeline Structural Multiplex Comparison BUD} (the pipeline for comparing two groups of subjects using structural multiplex data and binary undirected graphs at fixed densities) and adapt it in order to add a third group for structural multiplex.